src/App.js

```diff
@@ -1,25 +1,34 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react"
+import Header from "./components/Header"
+import Footer from "./components/Footer"
+import CatEdit from "./pages/CatEdit"
+import CatIndex from "./pages/CatIndex"
+import CatNew from "./pages/CatNew"
+import CatShow from "./pages/CatShow"
+import Home from "./pages/Home"
+import NotFound from "./pages/NotFound"
+import { Routes, Route } from "react-router-dom"
+import "./App.css"
+import mockCat from "./mockCat.js"
 
-function App() {
+const App = () => {
+  const [cats, setCats] = useState(mockCat)
+
+  console.log(cats)
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    <>
+    <header><Header /></header>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/catindex" element={<CatIndex />} />
+        <Route path="/catshow" element={<CatShow />} />
+        <Route path="/catnew" element={<CatNew />} />
+        <Route path="/catedit" element={<CatEdit />} />
+        <Route path="*" element={<NotFound />} />
+      </Routes>
+      <footer><Footer /></footer>
```

Comment:
Your component calls for Footer and Header don't need to be wrapped in a tag. If you use the header and footer tags, they would live in the actual Footer and Header component. Both of those tags function like div tags. They are just container tags.